query/reverse.js

```diff
@@ -30,7 +31,7 @@ function generateQuery( clean ){
 
   // set size
   if( clean.size ){
-    vs.var( 'size', clean.size );
+    vs.var( 'size', calcSize(clean.size));
```

Comment:
is it possible to call the `calcSize()` function as middleware or part of the sanitization steps?

it would be cool if the queries were not aware of any specific functionality and just needed to know about `clean`, it would mean that we could be free to enable/disable middleware without also having to change the queries.